components/script_traits/script_msg.rs

```diff
@@ -59,6 +60,9 @@ pub enum LogEntry {
 /// Messages from the script to the constellation.
 #[derive(Deserialize, Serialize)]
 pub enum ScriptMsg {
+    /// Broadcast a storage event to every same-origin pipeline.
+    /// The strings are url, key, old value and new value.
+    BroadcastStorageEvent(PipelineId, StorageType, String, Option<String>, Option<String>, Option<String>),
```

Comment:
I agree with you, but we should be consistent about it, not just adopt struct variants piecemeal.